concent_api/core/views.py

```diff
@@ -1241,6 +1378,239 @@ def store_message_and_message_status(
         receive_message_status.save()
 
 
+def store_subtask(
+    task_id:                        str,
+    subtask_id:                     str,
+    provider_public_key:            bytes,
+    requestor_public_key:           bytes,
+    state:                          Enum,
+    next_deadline:                  int,
+    task_to_compute:                message.TaskToCompute                = None,
+    report_computed_task:           message.ReportComputedTask           = None,
+    ack_report_computed_task:       message.AckReportComputedTask        = None,
+    reject_report_computed_task:    message.RejectReportComputedTask     = None,
+    subtask_results_accepted:       message.tasks.SubtaskResultsAccepted = None,
+    subtask_results_rejected:       message.tasks.SubtaskResultsRejected = None,
+):
```

Comment:
It would be good to add some comment here.

Reply:
@rwrzesien, what kind of comment would you expect? ;)

Reply:
What this function exactly does.

Reply:
:+1: